docs/source/changes/198.add_support_executors_batchsystem_adapters.yaml

```diff
@@ -0,0 +1,8 @@
+category: added
+summary: "Added support for executors in batchsystem adapters"
```

Comment:
`batch system adapters` instead of `batchsystem adapters`?

Reply:
Done!